### PR TITLE
Add release notes for 0.9.3

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

File: docs/editors/vscode.md [hunk 1]
```diff
@@ -196,8 +196,6 @@ guide them. In the end users should end up with something like this:
       "name": "Test class",
       // full name of the class to run
       "testClass": "com.example.Test",
-      // optional jvm properties to use
```
Comment (tgodzik) [inline docs/editors/vscode.md:199]: This was actually not true, there is nothing we can customize about test yet.

**Review by kpbochenek — APPROVED** [review]
LGTM! 

awesome to see worksheets for scala3 in :+1:

File: website/blog/2020-08-19-lithium.md [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+
+- hover
+- completions
+- go to definition
+- parameter hints
+
+The full list of supported features is available
+[here](../docs/editors/overview.html#additional-file-types).
+
+Sbt support is is achived by generating additional `metals.sbt` file for each
```
Comment (ckipp01) [inline website/blog/2020-08-19-lithium.md:69]:
```suggestion
sbt support is is achived by generating additional `metals.sbt` file for each
```

File: website/blog/2020-08-19-lithium.md [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+level of Sbt's project. For `build.sbt` we now generate
```
Comment (ckipp01) [inline website/blog/2020-08-19-lithium.md:70]:
```suggestion
level of sbt's project. For `build.sbt` we now generate
```

File: website/blog/2020-08-19-lithium.md [hunk 4]
```diff
@@ -0,0 +1,329 @@
...
+`project/project/metals.sbt`, for `project/other.sbt`
+`project/project/project/metals.sbt` etc. This is needed for the Bloop plugin to
+generate proper information in `.bloop` configuration files. It's recommended to
```
Comment (ckipp01) [inline website/blog/2020-08-19-lithium.md:73]:
```suggestion
properly generate information in the `.bloop` configuration files. It's recommended to
```

File: website/blog/2020-08-19-lithium.md [hunk 5]
```diff
@@ -0,0 +1,329 @@
...
+gitignore `metals.sbt` and `.bloop` anywhere in the file tree.
+
+## Scala 3 worksheet support
+
+Finally, it is possible to run worksheets for Scala 3 starting with version
```
Comment (ckipp01) [inline website/blog/2020-08-19-lithium.md:78]:
```suggestion
Finally, it is now possible to run worksheets for Scala 3 starting with version
```

File: website/blog/2020-08-19-lithium.md [hunk 6]
```diff
@@ -0,0 +1,329 @@
...
+0.26.0-RC1. To try it out users need to create the worksheet inside of a Scala 3
+source directory and can use it the same way as with previously supported
```
Comment (ckipp01) [inline website/blog/2020-08-19-lithium.md:80]:
```suggestion
source directory and then they can be used the same way as with previously supported
```

File: website/blog/2020-08-19-lithium.md [hunk 7]
```diff
@@ -0,0 +1,329 @@
...
+This is a great place for anyone to experiment with Scala 3 and see what is
+coming around the corner!
+
+![worksheet-sample](https://i.imgur.com/DZvz3Le.gif)
+
+## Running main classes from dependencies
+
+Previously, users could only run main classes that are defined in the source
+code, which made it impossible to run and debug in certain frameworks such for
+example [Play Framework](https://www.playframework.com/). Metals now searches
```
Comment (ckipp01) [inline website/blog/2020-08-19-lithium.md:92]:
```suggestion
code, which made it impossible to run and debug in certain frameworks such as the [Play Framework](https://www.playframework.com/). Metals now searches
```

File: website/blog/2020-08-19-lithium.md [hunk 8]
```diff
@@ -0,0 +1,329 @@
...
+In other editors users need to provide the same parameters for the run
+configuration.
+
+## Miscellaneous
+
+- Removed breakpoints are no longer hit.
+- Compilation is invoked only once for any change.
+- Worksheets now display types next to the code.
+- Fixed wrong completions in string interpolation.
+- Bloop now copies old artifacts by default even if workspace does not compile.
```
Comment (ckipp01) [inline website/blog/2020-08-19-lithium.md:126]:
```suggestion
- Bloop now copies old artifacts by default even if the workspace does not compile.
```

**Review by ckipp01 — CHANGES_REQUESTED** [review]
Just a few nits!

File: website/blog/2020-08-19-lithium.md [hunk 9]
```diff
@@ -0,0 +1,329 @@
...
+
+Thanks to the great work by [dos65](https://github.com/dos65) Metals now
+supports Sbt script files as well as build's Scala files.
+
+This includes in addition to already existing features:
+
+- hover
+- completions
+- go to definition
+- parameter hints
```
[the remark this answers is not in the document]
Comment (tgodzik) [reply website/blog/2020-08-19-lithium.md:64]: Good idea!

**Comment by tgodzik** [comment]
> Just a few nits!

Thanks, fixed them all now!

**Review by ckipp01 — APPROVED** [review]
Looks good! LGTM! 🚀

File: docs/contributors/releasing.md [hunk 10]
```diff
@@ -33,14 +33,17 @@ title: Making a release
...
   - Copy `website/blog/2018-12-06-iron.md` as a template
-  - You might use `.bin/release_notes.sc` script to generate merged PRs list. It
-    can be run using ammonite:
+  - You might use `./bin/merged_prs.sc` script to generate merged PRs list
+    between two last release tags. It can be run using ammonite:
 
   ```
   cs install ammonite
-  amm bin/release_notes.sc
+  amm ./bin/merged_prs.sc <tag1> <tag2> "<github_api_token>"
```
Comment (tgodzik) [inline docs/contributors/releasing.md:41]: I simplified the usage a bit.

**Pull request merged**